src/components/userFeedback/UserFeedback.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { extend } from 'underscore';
+
+export interface IUserFeedbackProps {
+  feedbackText: string;
+  state: string;
+  extraClasses?: string;
+  displayOnShow?: string;
+}
+
+export interface IUserFeedbackStyle {
+  displayClass: string;
```

Comment:
I'm not sure of what you mean here. IUserFeedbackStyle is the final style object that is passed to the view on render. It seems that it should be required to me, it is on hidden by default, because by default the state is valid. The displayClass than change if the state becomes WARNING or ERROR.  Does that make sens? I've changed the `getDefaultUserFeedbackStyle`to `getUserFeedbackStyleOnStateValid` in case that would be more explicit and/or comprehensible...hope it is :S